platform-includes/getting-started-use/javascript.hono.mdx

```diff
@@ -1,23 +1,10 @@
 ```javascript
-import * as Sentry from "@sentry/node";
-import { Hono, HTTPException } from "hono";
+// Require this first!
+require("./instrument");
 
-const app = new Hono()
-  // Add an onError hook to report unhandled exceptions to Sentry.
-  .onError((err, c) => {
-    // Report _all_ unhandled errors.
-    Sentry.captureException(err);
-    if (err instanceof HTTPException) {
-      return err.getResponse()
-    }
-    // Or just report errors which are not instances of HTTPException
-    // Sentry.captureException(err);
-    return c.json({ error: "Internal server error" }, 500)
-  })
-  // Your routes...
-  app.get("/", () => {
-    // ...
-  });
+// Now require other modules
+const { serve } = require("@hono/node-server");
+const { Hono } = require("hono");
```

Comment:
~We should add the ESM-equivalent for this.~ I just saw, this is the "CommonJS" section - so this is actually fine :)

```js
import './instrument';
import { server } from "@hono/node-server"
import { Hono } from 'hono'
```